src/common/localStorage.js

```diff
@@ -1,27 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: MIT-0
-import { useState } from 'react';
```

Comment:
Split this file into two. One vanilla.js (used in `apply-mode.js`) and the other one is with the hook. 

This is needed for proper bundle management, `apply-mode.js` is a separate bundle